ambari/src/main/java/io/brooklyn/ambari/AmbariClusterImpl.java

```diff
@@ -342,6 +347,22 @@ private RecommendationWrapper getRecommendationWrapperFromAmbariHostGroups() {
                     .build());
         }
 
+        List<String> serverComponentsList = getConfig(AmbariCluster.SERVER_COMPONENTS);
+        if (!serverComponentsList.isEmpty()) {
+             HostGroup.Builder hostGroupBuilder = new HostGroup.Builder()
+                    .setName("server-group")
+                    .addComponents(serverComponentsList);
+            if (componentsByNode.containsKey("server-group")) {
+                hostGroupBuilder.addComponents(componentsByNode.get("server-group"));
+            }
+            blueprintBuilder.addHostGroup(hostGroupBuilder.build());
+
+            bindingsBuilder.addHostGroup(new HostGroup.Builder()
+                    .setName("server-group")
+                    .addHosts(ImmutableList.of(Iterables.getFirst(Entities.descendants(this, AmbariServer.class), null).getFqdn()))
```

Comment:
It would be better if the list of FQDNs was get from the list of `getAmbariServers()` to make sure that all servers are mapped.